ktor-client/ktor-client-ios/darwin/src/io/ktor/client/engine/ios/IosClientEngineConfig.kt

```diff
@@ -12,6 +12,7 @@ import platform.Foundation.*
  */
 typealias ChallengeHandler = (
```

Comment:
This is the disadvantage of properties with functional types: we can't simply add a parameter without source breaking changes :( Looks like we should always use functions with labda instead of plain properties

E.g. 
```kotlin
class MyConfig {
    var handler: () -> Unit = {}
}
```

should be always

```kotlin
class MyConfig {
    private var handler: () -> Unit = {}

    fun handler(block: () -> Unit) {
        handler = block
    }
}
```

Reply:
This param was never been released in the past

Reply:
Btw I’ll replace it with the method